Content.Server/Chemistry/Components/SolutionManager/RefillableSolutionComponent.cs

```diff
@@ -21,5 +22,21 @@ public class RefillableSolutionComponent : Component
         [DataField("solution")]
         public string Solution { get; set; } = "default";
 
+        /// <summary>
+        /// The maximum amount that can be transferred to the solution at once
+        /// </summary>
+        [DataField("maxrefill")]
```

Comment:
use camelCase (i.e., "maxRefill") for data-fields. Some older bits of code don't always do that.

Reply:
my bad, fixed.